models/database.go

```diff
@@ -371,6 +371,28 @@ var databaseSchema = [][]string{
 			PRIMARY KEY (id)
 		)`,
 	},
+
+	23: {
+		`CREATE TABLE ia_templates (
+			name VARCHAR NOT NULL,
+			version INTEGER NOT NULL,
+			summary VARCHAR NOT NULL,
+			tiers JSONB,
+			expr VARCHAR NOT NULL,
+			params JSONB,
+			"for" BIGINT,
+			severity VARCHAR NOT NULL,
+			labels TEXT,
+			annotations TEXT,
```

Comment:
Why TEXT and not JSONB? Can it be NOT NULL? Should it?

Reply:
Other models use `TEXT` for labels, so I used the same approach.
https://github.com/percona/pmm-managed/blob/c2cd5d57da2f4dccfe7b2694e1f75e0f53125ebd/models/database.go#L100

`alert.Rule` validation allows empty labels and annotations, so models follow common validness rules.

Reply:
If we are going to store JSON anyway, we probably may use JSONB and `{}` for empty set of labels. NULLs are less pleasant to work with

But no strong opinion from me there

Reply:
At first, it was JSONB, I change it to TEXT after that comment https://github.com/percona/pmm-managed/pull/541#discussion_r532482651

Will change to JSONB

Reply:
Ah, I remembered, we can use `JSOB` + Custom type with `Scan/Value` methods, **OR** we can use `TEXT` + `setStringMap`(will rename it) that returns byte slice.

Reply:
@AlekSi so we should choose something

Reply:
Ok, let's use `TEXT`